.phan/plugins/InlineVarPlugin.php

```diff
@@ -0,0 +1,203 @@
+<?php
```

Comment:
Can we avoid forking this here? I would prefer any of the other options you mentioned than maintain this

Reply:
Removed - this seems unnecessary